File: 获取设备中的所有字体/获取设备中的所有字体/ViewController.m
//
//  ViewController.m
//  获取设备中的所有字体
//
//  Created by maiyun on 15/6/16.
//  Copyright (c) 2015年 黄成都. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    for (NSString *familyName in [UIFont familyNames]) {
        NSLog(@"字体族：%@",familyName);
        for (NSString *fontName in [UIFont fontNamesForFamilyName:familyName]) {
            NSLog(@"\t字体：%@",fontName);
        }
    }
    
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: swift代码段/1swift初见/1swift初见/ViewController.swift
//
//  ViewController.swift
//  1swift初见
//
//  Created by 黄成都 on 15/8/16.
//  Copyright (c) 2015年 黄成都. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        print("===========================================")
        let test = Square(sideLength: 5.2, name: "my test square")
        let area = test.area()
        print("\n")
        print(area)
        print("\n")
        print(test.simpleDescription())
        print("\n")
        print("===========================================")
        var triangle = EquilateralTriangle(sideLength: 3.1, name: "a griangle")
        print(triangle.perimeter)
        print("\n")
        triangle.perimeter = 9.9
        print(triangle.sideLength)
        print("\n")
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

